### ISSUE TITLE
Changing an actor's class through the sheet selection changes it for every actor.

### ISSUE THREAD

**Issue by algnc** [cyface/foundryvtt-dcc#44]
I create 2 PCs - one named Warrior, one named Cleric. When I change the Cleric to Cleric, Warrior is also a Cleric. If I change Warrior to Warrior, Cleric becomes a Warrior. This is with nightly code downloaded 8/17 at 9AM.

**Comment by cyface** [comment]
@algnc Thanks for the report, Christian! @mooped can you take a look? I'm wondering if the character sheet preference is not per-character right now. :)

**Comment by mooped** [comment]
@cyface We had a quick chat with @algnc and confirmed it's down to Foundry's UI for setting the sheet class making it very easy to accidentally change the default sheet class instead of the current actor's sheet class. Unfortunately it's easy to do and I had a bit of a panic first time I made this mistake!

That UI is part of the core of Foundry, but maybe there's a way to hook into it and make it more obvious.

**Comment by cyface** [comment]
@mooped Ah, right. I had the same issue. I assume that the sheet choice is just stored in the DB, so maybe we just make a drop-down on the sheet itself for the class, rather than using the Foundry UI?

**Comment by mooped** [comment]
Yep, I'll take a look when I have a moment. That would solve the issue with the UI showing the unlocalised class names also. :)

**Comment by mooped** [comment]
I need to fix the styling, but I've managed to override the sheet config dialog without too much duplication (basically the template and an override for the EntitySheetConfig class that changes the template and adds the tab controller).

I need to tweak the styling before I push anything.

Since I was tweaking the template I also set it up to localise the class names.

![custom-sheet-select](https://user-images.githubusercontent.com/371357/90446438-477b2800-e0d9-11ea-9e64-34843b3ee6a2.png)

![default-sheet-select](https://user-images.githubusercontent.com/371357/90446447-4ba74580-e0d9-11ea-91e7-5e3f55e80ee5.png)

This tabbed setup should make it a much more deliberate option to change the default sheet class.

We might still need to look into a way of doing it from the sheet though - I don't think the header button to open this dialog is available to players.
